### DISCUSSION TITLE
Can I use multiple strips on an ESP32?

### DISCUSSION THREAD

**Discussion by ghost** [Makuna/NeoPixelBus#529] [Q&A] [answered]
Can I use multiple strips on an ESP32?
I have a dev board (ESP-VROOM-32) and I have three strips connected. My code looks like this:
```
NeoPixelBus<NeoGrbFeature, Neo800KbpsMethod> interiorStrip(PixelCount, 12);
NeoPixelBus<NeoGrbFeature, Neo800KbpsMethod> wave1Strip(PixelCount, 16);
NeoPixelBus<NeoGrbFeature, Neo800KbpsMethod> wave2Strip(PixelCount, 17);
```
In my setup() I do this:
```
  interiorStrip.Begin();
  interiorStrip.Show();
  wave1Strip.Begin();
  wave1Strip.Show();
  wave2Strip.Begin();
  wave2Strip.Show();
```
When I send commands to a strip they work perfectly, but instead of using different strips they always use the last one (waveStrip2) for everything.
Should this work? What did I miss?
Thanks
```

**Comment by Makuna** [comment, marked as answer]
This wiki entry contains some information on this.  https://github.com/Makuna/NeoPixelBus/wiki/ESP32-NeoMethods
Due to the hardware requirement, you can't instance the exact same "method" as it will reuse the same hardware feature to send, thus what you are experiencing.

Try NeoEsp32Rmt0800KbpsMethod , NeoEsp32Rmt1800KbpsMethod , NeoEsp32Rmt2800KbpsMethod for each of your NeoPixelBus and it will work.

**Reply by ghost** [reply]
Thanks. That makes sense.
